vagrant-pxe-airgap-harvester/README.md

```diff
@@ -0,0 +1,119 @@
+  - `sudo usermod -a -G libvirt $(whoami)`
+  - `sudo vim /etc/libvirt/libvirtd.conf`:
+    - `unix_sock_group = "libvirt"`
+    - `unix_sock_rw_perms = "0770"`
+    - `sudo systemctl restart libvirtd.service`
+    - ensuring logging out and logging back in
+    - `ansible-galaxy collection install community.general`
+    - `vagrant plugin install vagrant-libvirt`
+    - ensuring a `known_hosts` file is present in `~/.ssh/known_hosts` for the sake of the provisioning leveraging `sshpass`, so possible creation could be something like `touch ~/.ssh/known_hosts` if not present
+- then finally running `./setup_harvester.sh -c` , provided any edits have been made as desired to the `settings.yml` file prior to kicking off the script
+
+
+
+Prerequisites
+-------------
+
+-   ansible-base >= 2.10.0 & ansible-core >= 2.11.0
+-   Vagrant \>= 2.0.3.
+-   vagrant-libvirt plugin \>= 0.0.43 and \<= 0.8.2 (NOTE: Anything above 0.8.2 currently breaks the Vagrantfile loadout with libvirt)
+-   KVM (i.e. qemu-kvm), preferably the latest and greatest. This
+    environment was tested with qemu-kvm 2.11.
+-   Host with at least 16 CPU, 64GB RAM, and 500GB free disk space. (note: running this is **very** resource intensive depending on the number of nodes)
+-   To run with Rancher either airgapped or not airgapped you need to have installed `sshpass` \>=1.06-1
+-   Ansible Galaxy's Community General module, `ansible-galaxy collection install community.general` , the community.general module must be installed
+-   Libvirt 'default' network will need to be enabled / autostarted [libvirt default network](https://wiki.libvirt.org/Networking.html#id2)
+
+Quick Start
+-----------
+1. You can edit the `settings.yml` to change:
+    - `rancher_config.run_single_node_rancher` to `true` to enable running a single node non-airgapped Rancher instance, if you would like to airgap that Rancher instance you would set `rancher_config.run_single_node_air_gapped_rancher` to `true` as well
+    - `node_disk_size` for `rancher_config` is thin provisioned (so it won't truly take up the entire space), but if running air-gapped would need a minimum of 300G
+    - other options should be self documenting
+2. Then you can run `./setup_harvester.sh` **NOTE:** you more than likely will want to pass the flag `-c` or `--clean-ssh-known-hosts` so that when the additional configuration on each Harvester node runs after standing up Rancher old-host information is cleaned up, ex: `./setup_harvester.sh -c`, since cleaning the known_hosts file is a local file, the flag is optional, because you may want to manipulate that local file yourself instead of having this script manipulate that local file
+3.  You then can navigate to `https://<harvester_vip>:30443` to access the harvester UI.
```

Comment:
I think, it is important to add the below note here.
`NOTE: by default harvester_vip is 192.168.2.131. However, it is configureable in settings.yml.`

Reply:
sounds good :smile: -updated